src/components/atoms/ChatPoll/ChatPoll.scss

```diff
@@ -0,0 +1,126 @@
+@import "scss/constants";
+
+$counterparty-text-border-radius: 28px 28px 28px 4px;
+$my-text-border-radius: 28px 28px 4px 28px;
+
+.poll {
+  align-self: flex-start;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  margin-bottom: 16px;
+  width: 100%;
+
+  &__topic {
+    text-align: center;
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+
+  &__question {
+    display: block;
+    height: auto;
+    width: auto;
+    background: none;
+    border: none;
+    color: #fff;
+    font-size: 14px;
+    font-weight: initial;
+    padding: 0;
+    margin-bottom: 16px;
+    &:focus {
+      outline: none;
+    }
+  }
+
+  &__details {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    p {
+      margin: 0 10px 0 0;
+    }
```

Comment:
To expand on @mike-lvov's comment. As part of BEM, we style classes, not direct html elements. So instead of targeting `p`, we should make a `&__foo` or similar 'BEM element class' for it:

- http://getbem.com/naming/

Reply:
fixed